content/source/docs/enterprise/saml/identity-provider-configuration-okta.html.md

```diff
@@ -10,27 +10,30 @@ Follow these steps to configure Okta as the identity provider (IdP) for Terrafor
 ## Configure a New Okta SAML Application
 
 1. In Okta's web interface, go to the "Applications" tab and click "Create New App".
-  ![Screenshot: Okta's "Add Application" page](./images/sso-okta-add-application.png)
+    ![Screenshot: Okta's "Add Application" page](./images/sso-okta-add-application.png)
 2. Select "Web" as the platform and "SAML 2.0" as the sign on method, then click "Create".
-  ![Screenshot: Okta's modal dialog for beginning a new application](./images/sso-okta-new-application-form.png)
+    ![Screenshot: Okta's modal dialog for beginning a new application](./images/sso-okta-new-application-form.png)
 3. In the "General Settings" page, enter an app name, then click "Next".
-  ![Screenshot: the first page of Okta's new app configuration workflow, called "General Settings"](./images/sso-okta-new-application-general-settings.png)
+    ![Screenshot: the first page of Okta's new app configuration workflow, called "General Settings"](./images/sso-okta-new-application-general-settings.png)
 4. In the "Configure SAML" page, configure the following settings with the specified values:
-    - **Single sign on URL:** `https://<TFE HOSTNAME>/users/saml/auth` (listed as "ACS consumer (recipient) URL" in TFE's SAML settings).
-    - **Use this for Recipient URL and Destination URL** (checkbox): enabled.
+
+    -> **Note:** The SSO URL is used for authentication and is different from the [Login URL][login_url].  The [Login URL][login_url] will start the SSO Session utilizing this SSO URL behind-the-scenes.  To sign into TFE using Okta after everything is configured use the [Login URL][login_url].
+    - **SSO (Single Sign-On) URL:** `https://<TFE HOSTNAME>/users/saml/auth` (listed as "ACS consumer (recipient) URL" in TFE's SAML settings).
```

Comment:
```suggestion
    - **SSO (Single Sign-On) URL:** `https://<TFE HOSTNAME>/users/saml/auth` (listed as "ACS consumer (recipient) URL" in TFE's SAML settings).

        -> **Note:** The SSO URL is used by the identity provider software during authentication; it is different from the [Login URL][login_url], which users visit in order to sign in to Terraform Enterprise.
```

- Moving this callout down so that it's tied to the bullet point that references that URL. 
- Shrinking the description a bit.

Reply:
Thank you for desc compression!  Moved.